test/parallel/test-btoa-atob.js

```diff
@@ -15,3 +15,17 @@ throws(() => buffer.btoa(), /TypeError/);
 
 strictEqual(atob(' '), '');
 strictEqual(atob('  YW\tJ\njZA=\r= '), 'abcd');
+
+strictEqual(atob(null), '\x9Eée');
+strictEqual(atob(NaN), '5£');
+strictEqual(atob(Infinity), '"wâ\x9E+r');
+strictEqual(atob(true), '¶»\x9E');
+strictEqual(atob(1234), '×mø');
+strictEqual(atob([]), '');
+strictEqual(atob({ toString: () => '' }), '');
+strictEqual(atob({ [Symbol.toPrimitive]: () => '' }), '');
+
+throws(() => atob(), /ERR_MISSING_ARGS/);
```

Comment:
There’s already a test for that just above. 
```suggestion
```

Reply:
Thanks @aduh95. Thoughts on updating the assertion to be a bit more specific?

Something like:

```js
throws(() => atob(), {
  name: 'TypeError',
  message: 'The "input" argument must be specified',
  code: 'ERR_MISSING_ARGS'
});
```

Reply:
Not really, we want this API to stick to the web one, so we only car if it throws a `TypeError` in this case (because the web throws a `TypeError`, not an error with a `code` property). You'd be correct for any other API though.
_On a side note,  we try to avoid having test cases for error `message` as those are not covered by the semver rule (meaning it's not considered a breaking change to update an error message)._

Reply:
Super helpful. Thanks so much for the detailed response. I'll keep that in mind when contributing in the future!